pkg/asset/rhcos/image.go

```diff
@@ -94,20 +94,40 @@ func osImage(ctx context.Context, config *types.InstallConfig) (string, string,
 		if len(config.Platform.AWS.AMIID) > 0 {
 			return config.Platform.AWS.AMIID, "", nil
 		}
+		const globalResourceRegion = "us-east-1"
+		var osimageControlPlane, osimageCompute string
 		region := config.Platform.AWS.Region
-		if !rhcos.AMIRegions(config.ControlPlane.Architecture).Has(region) {
-			const globalResourceRegion = "us-east-1"
-			logrus.Debugf("No AMI found in %s. Using AMI from %s.", region, globalResourceRegion)
-			region = globalResourceRegion
+		{
```

Comment:
Is this { } required? Seems like there is no conditional here.

Reply:
Done